setup.py

```diff
@@ -34,6 +34,7 @@
         'delfin.storage.drivers': [
             'fake_storage fake_driver = delfin.drivers.fake_storage:FakeStorageDriver',
             'dellemc vmax = delfin.drivers.dell_emc.vmax.vmax:VMAXStorageDriver',
+            'hpe threeparstor = delfin.drivers.hpe.hpe_3par.hpe_3parstor:Hpe3parStorDriver',
```

Comment:
'threeparstor' doesn't look good to me how about just '3par' or '3par_storeserv' ?